src/components/input/input.js

```diff
@@ -140,6 +140,8 @@ function labelDirective() {
       if (!containerCtrl || attr.mdNoFloat || element.hasClass('_md-container-ignore')) return;
 
       containerCtrl.label = element;
+      containerCtrl.element.toggleClass('md-input-has-label', true);
```

Comment:
This one's also the equivalent of `containerCtrl.element.addClass('md-input-has-label');`